objective.py

```diff
@@ -8,8 +8,8 @@ class Objective(BaseObjective):
     name = "Lasso Regression"
 
     parameters = {
-        'reg': [0.05, .1, .5],
-        'fit_intercept': [True, False]
+        'fit_intercept': [True, False],
+        'reg': [.5, .1, .05],
     }
```

Comment:
Not sure why this change was made but this broke the documentation on `benchopt` as an implicit assumption was made on the order of the parameters.

`benchopt/benchopt#265` fixes this by ensuring a deterministic order for the parameters.

Reply:
Thanks ! Random changed happened after tweaking objective configurations for debug

Reply:
As it was unsafe, I guess this is good as this is now fixed ;)